projects/storefrontlib/cms-components/product/product-intro/product-intro.component.ts

```diff
@@ -28,6 +34,11 @@ export class ProductIntroComponent {
   product$: Observable<Product | null> =
     this.currentProductService.getProduct();
 
+  //TODO: (CXSPA-8044) - remove feature flag next major release.
+  @Optional() protected featureConfigService = inject(FeatureConfigService, {
+    optional: true,
+  });
+
```

Comment:
```suggestion
protected featureConfigService = inject(FeatureConfigService);
```

We've changed our approach slightly recently. The ticket number in the comment used to refer to a task reminding us to remove the feature flag eventually. Those have been deemed unnecessary and are no longer used.

Reply:
I've updated.